PancakeSwap-Bots/Prediction-Admin-Operatons/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "prediction-admin-operations-bot",
+  "version": "0.0.1",
+  "description": "Detects Admin and Operator Operations in PancakePredictionV2 ",
+  "repository": "https://github.com/NethermindEth/Forta-Agents/tree/main/PancakeSwap-Bots/Prediction-Admin-Operatons",
+  "chainIds": [
+    56
+  ],
+  "scripts": {
+    "format": "prettier --write \"src/**/*.ts\"",
+    "build": "tsc",
+    "start": "npm run start:dev",
+    "start:dev": "nodemon --watch src --watch forta.config.json -e js,ts,json  --exec \"npm run build && forta-agent run\"",
+    "start:prod": "forta-agent run --prod",
+    "tx": "npm run build && forta-agent run --tx",
+    "block": "npm run build && forta-agent run --block",
+    "range": "npm run build && forta-agent run --range",
+    "file": "npm run build && forta-agent run --file",
+    "publish": "forta-agent publish",
+    "info": "forta-agent info",
+    "logs": "forta-agent logs",
+    "push": "forta-agent push",
+    "disable": "forta-agent disable",
+    "enable": "forta-agent enable",
+    "keyfile": "forta-agent keyfile",
+    "test": "jest"
+  },
+  "dependencies": {
+    "forta-agent": "^0.1.8"
+  },
+  "devDependencies": {
+    "@types/jest": "^27.0.1",
+    "@types/nodemon": "^1.19.0",
+    "jest": "^27.0.6",
+    "nodemon": "^2.0.8",
```

Comment:
```suggestion
    "nodemon": "^2.0.19",
```
Updating this, as the old version contained some vulnerability warnings. (new `package-lock.json` also needs to be pushed)